### ISSUE TITLE
Trouble encrypting long ArrayBuffers (AES-CBC mode)

### ISSUE THREAD

**Issue by elsehow** [PeculiarVentures/node-webcrypto-ossl#79]
Hi!

signal-protocol relies on [AES in CBC mode to encrypt messages](https://github.com/elsehow/signal-protocol/blob/master/src/crypto.js#L38):

```javascript
crypto.subtle.encrypt({name: 'AES-CBC', iv: new Uint8Array(iv)}, key, data);
```

 When trying to encrypt a long `ArrayBuffer` (larger than `byteLength` 239), I find the encrypted output much shorter than expected. 

Here's a test that fails in Node with node-webcrypto-ossl, but works in the browser with the WebCrypto API:

[IntegrationTest.js L118-126](https://github.com/elsehow/signal-protocol/blob/master/test/IntegrationTest.js#L118-L126) 

(see also: accompanying [travis log](https://travis-ci.org/elsehow/signal-protocol/jobs/184849261))

Any idea what could be going on? Let me know how I can help!

**Comment by elsehow** [comment]
Investigating further, I find that 

```javascript
      return crypto.subtle.importKey('raw', key, {
          name: 'AES-CBC'
      }, false, ['encrypt'])
```

produces

```javascript
CryptoKey {
  usages: [ 'encrypt' ],
  native_: AesKey {},
  extractable: false,
  algorithm: { name: 'AES-CBC' },
  type: 'secret' }
```

In the browser, `algorithm` has an additional property `length`, which is 256. Not sure what this means yet.

**Comment by microshine** [comment]
@elsehow I published new version of `node-webcrypto-ossl` - v1.0.15. Thank you for your issue

**Comment by elsehow** [comment]
That fixes my issue! Thanks much
